azure-pipelines-official.yml

```diff
@@ -38,21 +38,26 @@ stages:
     - powershell: Write-Host "##vso[task.setvariable variable=SourceBranchName]$('$(Build.SourceBranch)'.Substring('refs/heads/'.Length))"
       displayName: Setting SourceBranchName variable
 
-    - powershell: Write-Host "##vso[task.setvariable variable=IsPRValidationBuild]($(PRNumber) -ne '') -and ($(PRNumber) -ne 'default')"
-      displayName: Setting IsPRValidationBuild variable
-
     - task: tagBuildOrRelease@0
-      displayName: Tag build
+      displayName: Tag official build
+      inputs:
+        type: 'Build'
+        tags: 'OfficialBuild'
+      condition: and(succeeded(), eq(variables['PRNumber'], 'default'))
```

Comment:
Would be nice if PRNumber could be '' or 'default', but hopefully people will just leave the variable 'default' when queuing.